### DISCUSSION TITLE
How to get Dynamic Filter Parameter

### DISCUSSION THREAD

**Discussion by fadliasyhariStar** [Budibase/budibase#12516] [Help] [answered]
I am using a data provider with a Custom Query on a Postgres data source. I want to create an export button that generates a query for the data based on applied filters. 
However, I am facing difficulty in obtaining dynamic filter parameters. Is there a method to retrieve the filter results?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @fadliasyhariStar,

You can't get the value of dynamic filters. I'm not sure how you've even integrated this with your custom query as there is no way to reference this elsewhere within Budibase. The dynamic filter component only filters on the frontend, I don't believe it can send requests to custom queries.

Feel free to create a feature request within issues if you feel this should be added to the core product.

Are you use that you aren't just using a text field component instead? If you are you could simply use the on change action attached to the field and save the value of this to persisted state. That could then be referenced anywhere you like.

I hope this makes sense.
